frontend/ui/lib/widgets/create_trip_widget.dart

```diff
@@ -1,21 +1,39 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:tripmeout/widgets/map_widget.dart';
+import 'package:tripmeout/services/trip_service.dart';
+import 'package:tripmeout/model/trip.dart';
+import 'package:tripmeout/model/location.dart';
+import 'package:tripmeout/router/router.dart';
 
 //TODO: Add loading screen after to the Create Trip Widget
 //TODO: Get rid of the newInformation text on the page.
 
 class CreateTripWidget extends StatefulWidget {
+  final TripService tripService;
+
+  CreateTripWidget(this.tripService);
+
   @override
-  _CreateTripWidgetState createState() => _CreateTripWidgetState();
+  _CreateTripWidgetState createState() =>
+      _CreateTripWidgetState(this.tripService);
```

Comment:
FYI, the state object has access to the containing widget via the `widget` property. This should probably be accessed that way instead off plumbing the property through manually.